S/W Version Information
Model: SM-R760
Tizen-Version: 2.3.2.3
Build-Number: R760XXU2BQC5
Build-Date: 2017.03.15 16:06:31

Crash Information
Process Name: recorder
PID: 21185
Date: 2018-04-23 16:01:42+0200
Executable File Path: /opt/usr/apps/es.ugr.frailty.frailtylauncher/bin/recorder
Signal: 6
      (SIGABRT)
      si_code: -6
      signal sent by tkill (sent by pid 21185, uid 5000)

Register Information
r0   = 0x00000000, r1   = 0x000052c1
r2   = 0x00000006, r3   = 0x000052c1
r4   = 0x00000006, r5   = 0x00000002
r6   = 0xf7111bec, r7   = 0x0000010c
r8   = 0x00000038, r9   = 0x00000002
r10  = 0xffb29238, fp   = 0xffb29850
ip   = 0xf67894c0, sp   = 0xffb29118
lr   = 0xf7013328, pc   = 0xf700f668
cpsr = 0x20000010

Memory Information
MemTotal:   714612 KB
MemFree:      9576 KB
Buffers:     60404 KB
Cached:     226132 KB
VmPeak:      12984 KB
VmSize:      12624 KB
VmLck:           0 KB
VmPin:           0 KB
VmHWM:        7904 KB
VmRSS:        7900 KB
VmData:       2652 KB
VmStk:         136 KB
VmExe:          12 KB
VmLib:        7936 KB
VmPTE:          40 KB
VmSwap:          0 KB

Maps Information
f678c000 f678f000 r-xp /lib/libattr.so.1.1.0
f6798000 f679f000 r-xp /lib/libcrypt-2.13.so
f67cf000 f67d2000 r-xp /lib/libcap.so.2.21
f67da000 f67dc000 r-xp /usr/lib/libiri.so
f67e4000 f6801000 r-xp /usr/lib/libsecurity-server-commons.so.1.0.0
f680a000 f680e000 r-xp /usr/lib/libsmack.so.1.0.0
f6818000 f681a000 r-xp /usr/lib/libgmodule-2.0.so.0.3200.3
f6822000 f6827000 r-xp /usr/lib/libffi.so.5.0.10
f682f000 f6830000 r-xp /usr/lib/libgthread-2.0.so.0.3200.3
f6838000 f683a000 r-xp /usr/lib/journal/libjournal.so.0.1.0
f6842000 f6844000 r-xp /usr/lib/libSLP-db-util.so.0.1.0
f684e000 f685e000 r-xp /lib/libresolv-2.13.so
f6862000 f687a000 r-xp /usr/lib/liblzma.so.5.0.3
f6882000 f6884000 r-xp /usr/lib/libsecurity-extension-common.so.1.0.1
f688c000 f68bb000 r-xp /usr/lib/libsecurity-server-client.so.1.0.1
f68c4000 f68d3000 r-xp /usr/lib/libjson-glib-1.0.so.0.1001.3
f68dc000 f68e6000 r-xp /usr/lib/libsensord-shared.so
f68ef000 f6923000 r-xp /usr/lib/libgobject-2.0.so.0.3200.3
f692c000 f69ff000 r-xp /usr/lib/libgio-2.0.so.0.3200.3
f6a0a000 f6a20000 r-xp /lib/libz.so.1.2.5
f6a28000 f6a2e000 r-xp /usr/lib/libcapi-base-common.so.0.1.8
f6a37000 f6aac000 r-xp /usr/lib/libsqlite3.so.0.8.6
f6ab6000 f6ad0000 r-xp /usr/lib/libpkgmgr_parser.so.0.1.0
f6ad8000 f6ade000 r-xp /lib/librt-2.13.so
f6ae7000 f6b05000 r-xp /usr/lib/libsystemd.so.0.4.0
f6b0f000 f6b10000 r-xp /usr/lib/libsecurity-extension-interface.so.1.0.1
f6b18000 f6b1d000 r-xp /usr/lib/libxdgmime.so.1.1.0
f6b25000 f6bf5000 r-xp /usr/lib/libglib-2.0.so.0.3200.3
f6bf6000 f6c20000 r-xp /usr/lib/libdbus-1.so.3.8.12
f6c29000 f6c40000 r-xp /usr/lib/libdbus-glib-1.so.2.2.2
f6c48000 f6cb1000 r-xp /lib/libm-2.13.so
f6cba000 f6d4e000 r-xp /usr/lib/libstdc++.so.6.0.16
f6d61000 f6d66000 r-xp /usr/lib/libctx-client.so.0.8.3
f6d6e000 f6d75000 r-xp /usr/lib/libctx-shared.so.0.8.3
f6d7d000 f6da7000 r-xp /usr/lib/libsensor.so.1.9.6
f6db0000 f6e7c000 r-xp /usr/lib/libxml2.so.2.7.8
f6e89000 f6e8b000 r-xp /usr/lib/libiniparser.so.0
f6e94000 f6e97000 r-xp /usr/lib/libbundle.so.0.1.22
f6e9f000 f6ea5000 r-xp /usr/lib/libappsvc.so.0.1.0
f6ead000 f6ed0000 r-xp /usr/lib/libpkgmgr-info.so.0.0.17
f6ed8000 f6f19000 r-xp /usr/lib/libeina.so.1.7.99
f6f22000 f6f39000 r-xp /usr/lib/libecore.so.1.7.99
f6f50000 f6f59000 r-xp /usr/lib/libvconf.so.0.2.45
f6f61000 f6f75000 r-xp /lib/libpthread-2.13.so
f6f80000 f6f8d000 r-xp /usr/lib/libaul.so.0.1.0
f6f97000 f6f99000 r-xp /lib/libdl-2.13.so
f6fa2000 f6fad000 r-xp /lib/libunwind.so.8.0.1
f6fda000 f6fe2000 r-xp /lib/libgcc_s-4.6.so.1
f6fe3000 f7107000 r-xp /lib/libc-2.13.so
f7115000 f7117000 r-xp /usr/lib/libdlog.so.0.0.0
f711f000 f712b000 r-xp /usr/lib/libcapi-system-sensor.so.0.1.19
f7134000 f7139000 r-xp /usr/lib/libcapi-system-info.so.0.2.0
f7141000 f7145000 r-xp /usr/lib/libcapi-appfw-app-control.so.0.3.2.5
f714e000 f7150000 r-xp /usr/lib/libcapi-appfw-app-common.so.0.3.2.5
f7159000 f715c000 r-xp /usr/lib/libappcore-agent.so.1.1
f717a000 f7197000 r-xp /lib/ld-2.13.so
f71a0000 f71a3000 r-xp /opt/usr/apps/es.ugr.frailty.frailtylauncher/bin/recorder
f71a3000 f71a7000 r-xp /usr/lib/libsys-assert.so
f7813000 f7889000 rw-p [heap]
ffb0a000 ffb2b000 rw-p [stack]
End of Maps Information

Callstack Information (PID:21185)
Call Stack Count: 2
 0: gsignal + 0x3c (0xf700f668) [/lib/libc.so.6] + 0x2c668
 1: (0x1) (null)
End of Call Stack

Package Information
Package Name: es.ugr.frailty.frailtylauncher
Package ID : es.ugr.frailty.frailtylauncher
Version: 1.0.0
Package Type: rpm
App Name: recorder
App ID: es.ugr.frailty.recorder
Type: capp
Categories: 

Latest Debug Message Information
--------- beginning of /dev/log_main
-23 16:01:42.911+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:42.911+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:42.911+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:42.921+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:42.921+0200 E/LOCATION(19762): location-privacy.c: location_get_app_type(54) > Fail to get app_id. Err[-1]
04-23 16:01:42.921+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:42.971+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:42.981+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:43.031+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(195) > Successfully de-registered(0)
04-23 16:01:43.031+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(295) > Successfully de-registered(0)
04-23 16:01:43.031+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(395) > Successfully de-registered(0)
04-23 16:01:43.031+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_destroy(471) > Destroy handle: 0xf7374110
04-23 16:01:43.091+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_create(453) > New handle created[0xf3e179a0]
04-23 16:01:43.091+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(184) > Successfully registered(1)
04-23 16:01:43.091+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(284) > Successfully registered(1)
04-23 16:01:43.091+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(384) > Successfully registered(1)
04-23 16:01:43.091+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_get_type(507) > Connected Network = 2
04-23 16:01:43.101+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:43.101+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:43.121+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:43.131+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:43.131+0200 E/LOCATION(19762): location-privacy.c: location_privacy_initialize(165) > Fail to get app_id. Err[-1]
04-23 16:01:43.131+0200 I/LOCATION(19762): location.c: location_new(269) > method: 1
04-23 16:01:43.131+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:43.131+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (gps) open success
04-23 16:01:43.131+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(remote_gps) is found
04-23 16:01:43.131+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (remote_gps) open success
04-23 16:01:43.131+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:43.131+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:43.131+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:43.141+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:43.151+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:43.151+0200 E/LOCATION(19762): location-privacy.c: location_get_app_type(54) > Fail to get app_id. Err[-1]
04-23 16:01:43.151+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:43.211+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:43.211+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(195) > Successfully de-registered(0)
04-23 16:01:43.211+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(295) > Successfully de-registered(0)
04-23 16:01:43.211+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(395) > Successfully de-registered(0)
04-23 16:01:43.211+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_destroy(471) > Destroy handle: 0xf3e179a0
04-23 16:01:43.691+0200 I/servicemanager(19708): es.ugr.frailty.location sleep timeout
04-23 16:01:43.691+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.location)
04-23 16:01:43.691+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:43.691+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 19708
04-23 16:01:43.701+0200 W/AUL_AMD ( 2476): amd_launch.c: start_process(606) > child process: 22613
04-23 16:01:43.701+0200 E/RESOURCED( 2584): block.c: block_prelaunch_state(138) > insert data es.ugr.frailty.frailtylauncher, table num : 1
04-23 16:01:43.751+0200 E/CAPI_APPFW_APPLICATION(22613): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:43.751+0200 E/CAPI_APPFW_APPLICATION(22613): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:43.761+0200 W/AUL_AMD ( 2476): amd_launch.c: __send_app_launch_signal(397) > send launch signal done: 22613
04-23 16:01:43.761+0200 W/AUL     ( 2476): app_signal.c: aul_send_app_launch_request_signal(521) > aul_send_app_launch_request_signal app(es.ugr.frailty.location) pid(22613) type(svcapp) bg(0)
04-23 16:01:43.761+0200 W/AUL_AMD ( 2476): amd_status.c: __socket_monitor_cb(1277) > (22613) was created
04-23 16:01:43.771+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(298) > request cmd(0) result(22613)
04-23 16:01:43.771+0200 I/servicemanager(19708): es.ugr.frailty.location stop request sent!
04-23 16:01:43.771+0200 I/servicemanager(19708): App control destroyed.
04-23 16:01:43.771+0200 W/STARTER ( 2701): pkg-monitor.c: _app_mgr_status_cb(395) > [_app_mgr_status_cb:395] Launch request [22613]
04-23 16:01:43.781+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:43.791+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22613
04-23 16:01:43.791+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:43.811+0200 W/LOCATION(22613): location-common-util.c: location_is_onboard_gps_use_only(180) > app_type [capp]
04-23 16:01:43.811+0200 E/LOCATION(22613): location-common-util.c: location_is_onboard_gps_use_only(191) > Failed to get metadata. Err[-2]
04-23 16:01:43.811+0200 E/PKGMGR_INFO(22613): pkgmgrinfo_appinfo.c: pkgmgrinfo_appinfo_destroy_appinfo(3554) > (handle == NULL) appinfo handle is NULL
04-23 16:01:43.811+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:43.811+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:43.821+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22613
04-23 16:01:43.831+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:43.841+0200 I/LOCATION(22613): location.c: location_new(269) > method: 0
04-23 16:01:43.841+0200 W/LOCATION(22613): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:43.851+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:43.861+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:43.871+0200 W/LOCATION(22613): module-internal.c: module_new(311) > module (gps) open success
04-23 16:01:43.871+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(remote_gps) is found
04-23 16:01:43.871+0200 W/LOCATION(22613): module-internal.c: module_new(311) > module (remote_gps) open success
04-23 16:01:43.871+0200 W/LOCATION(22613): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:43.871+0200 W/LOCATION(22613): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:43.871+0200 W/LOCATION(22613): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:43.901+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(wps) is found
04-23 16:01:43.911+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(wps) is found
04-23 16:01:43.911+0200 W/LOCATION(22613): module-internal.c: module_new(311) > module (wps) open success
04-23 16:01:43.921+0200 W/LOCATION(22613): module-internal.c: module_is_supported(340) > module name(remote_wps) is found
04-23 16:01:43.921+0200 W/LOCATION(22613): module-internal.c: module_new(311) > module (remote_wps) open success
04-23 16:01:43.921+0200 W/LOCATION(22613): location-wps.c: __update_remote_connection_status(215) > update FALSE
04-23 16:01:43.921+0200 W/LOCATION(22613): location-wps.c: __update_remote_connection_status(220) > is_remote_connected[0]
04-23 16:01:43.961+0200 W/LOCATION(22613): location-gps.c: location_gps_set_property(1651) > id [14], value [0]
04-23 16:01:43.961+0200 I/LOCATION(22613): location-gps.c: location_gps_set_property(1778) > Set prop>> hybrid_obj: 0xf73dbc18
04-23 16:01:43.961+0200 I/LOCATION(22613): location-gps.c: location_gps_set_property(1793) > Set prop>> is_hybrid: 1
04-23 16:01:43.961+0200 I/LOCATION(22613): location-wps.c: location_wps_set_property(1182) > Set prop>> hybrid_obj: 0xf73dbc18
04-23 16:01:43.961+0200 I/LOCATION(22613): location-wps.c: location_wps_set_property(1194) > Set prop>> is_hybrid: 1
04-23 16:01:43.961+0200 I/location(22613): es.ugr.frailty.location: creado servicio de localización
04-23 16:01:44.031+0200 I/LOCATION(22613): location-hybrid.c: location_hybrid_start(641) > IOS Connected [0]
04-23 16:01:44.101+0200 W/LOCATION(22613): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:44.101+0200 W/LOCATION(22613): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:44.101+0200 I/LOCATION(22613): location-gps.c: __get_availability_use_gps(508) > Check _availability use GPS
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHook: API: gpsSetPositionMode(GpsiHookStateIdle: e,c,g)
04-23 16:01:44.181+0200 W/libgps  ( 3123): proxy__gps_set_position_mode(0, 0, 1000, 50, 0)
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHook: API: gpsStart(GpsiHookStateIdle: e,c,g)
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHookStateIdle             : EXIT
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHookStateGps              : ENTRY
04-23 16:01:44.181+0200 W/libgps  ( 3123): [proxy__gps_start][line = 1022]: called.
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHook: API: gpsStatusCb(GpsiHookStateGps: e,c,g)
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHook: API: gpsStatusCb(GpsiHookStateGps: e,c,g)
04-23 16:01:44.181+0200 W/gpsd    ( 3104): HandleIncomingMessage() dest id:2, msg id:26
04-23 16:01:44.181+0200 D/gpsd_d  ( 3104): SetIgnoreAsstMask: ignore_asst_mask = 0x00000000
04-23 16:01:44.181+0200 W/gpsd    ( 3104): HandleIncomingMessage() dest id:2, msg id:27
04-23 16:01:44.181+0200 W/gpsd    ( 3104): HandleIncomingMessage() dest id:0, msg id:8
04-23 16:01:44.181+0200 W/libgps_d( 3123): OnGpsExtensionMessage: message_id(2), data((nil)), size(0)
04-23 16:01:44.181+0200 W/gpsd    ( 3104): HandleIncomingMessage() dest id:0, msg id:1
04-23 16:01:44.181+0200 W/libgps_d( 3123): OnGpsExtensionMessage: message_id(11), data(0xf78b8138), size(4)
04-23 16:01:44.181+0200 W/libgps_d( 3123): [OnGpsExtensionMessage] Received: MSG_SET_LOG_LEVEL(2)
04-23 16:01:44.181+0200 W/libgps_d( 3123): OnGpsExtensionMessage: message_id(6), data((nil)), size(0)
04-23 16:01:44.181+0200 W/libgps_d( 3123): GpsiHook: API: gpsRequestUtcTimeCb(GpsiHookStateGps: e,c,g)
04-23 16:01:44.261+0200 W/LOCATION(22613): location-wps.c: __update_remote_connection_status(215) > update FALSE
04-23 16:01:44.261+0200 W/LOCATION(22613): location-wps.c: __update_remote_connection_status(220) > is_remote_connected[0]
04-23 16:01:44.361+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:44.371+0200 W/libgps_d( 3123): OnGpsExtensionMessage: message_id(1), data(0xf78b8138), size(2752)
04-23 16:01:44.391+0200 W/libgps_d( 3123): GpsiHook: API: gpsLocationCb(GpsiHookStateGps: e,c,g)
04-23 16:01:44.391+0200 W/libgps_d( 3123): GpsiHook: API: gpsSvStatusCb(GpsiHookStateGps: e,c,G)
04-23 16:01:44.391+0200 W/libgps  ( 3123): proxy__gps_sv_status_cb : called
04-23 16:01:44.411+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_create(453) > New handle created[0xf736d4d0]
04-23 16:01:44.411+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(184) > Successfully registered(1)
04-23 16:01:44.411+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(284) > Successfully registered(1)
04-23 16:01:44.411+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(384) > Successfully registered(1)
04-23 16:01:44.411+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_get_type(507) > Connected Network = 2
04-23 16:01:44.421+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:44.421+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:44.421+0200 W/libgps_d( 3123): OnGpsExtensionMessage: message_id(4), data((nil)), size(0)
04-23 16:01:44.421+0200 W/libgps  ( 3123): proxy__agps_ril_request_refloc_cb: refLocationSaved(1), time1(6000082), time2(5997887)
04-23 16:01:44.421+0200 W/libgps  ( 3123): proxy__agps_ril_set_ref_location: agps_reflocation->type(0), refLocationInfo.refLocationSaved(1)
04-23 16:01:44.421+0200 W/libgps  ( 3123): proxy__agps_ril_set_ref_location: type(0), mcc(0), mnc(0), cid(0), psc(0), lac(0) 
04-23 16:01:44.421+0200 I/libgps_d( 3123): OnGpsExtensionMessage: MSG_REQUEST_LOCATION_ID CELL_ID
04-23 16:01:44.421+0200 W/gpsd    ( 3104): HandleIncomingMessage() dest id:2, msg id:25
04-23 16:01:44.431+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:44.431+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:44.431+0200 E/LOCATION(19762): location-privacy.c: location_privacy_initialize(165) > Fail to get app_id. Err[-1]
04-23 16:01:44.431+0200 I/LOCATION(19762): location.c: location_new(269) > method: 1
04-23 16:01:44.431+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:44.431+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (gps) open success
04-23 16:01:44.431+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(remote_gps) is found
04-23 16:01:44.431+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (remote_gps) open success
04-23 16:01:44.431+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:44.431+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:44.431+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:44.441+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:44.451+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:44.451+0200 E/LOCATION(19762): location-privacy.c: location_get_app_type(54) > Fail to get app_id. Err[-1]
04-23 16:01:44.451+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:44.501+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:44.511+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:44.521+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:44.521+0200 E/LOCATION(19762): location-privacy.c: location_get_privacy(232) > Fail to get app_id. Err[-1]
04-23 16:01:44.521+0200 I/LOCATION(19762): location-privacy.c: location_get_privacy(233) > Skip check because it might be daemon process
04-23 16:01:44.521+0200 I/LOCATION(19762): location-gps.c: __get_availability_use_gps(508) > Check _availability use GPS
04-23 16:01:44.551+0200 W/LOCATION(22613): location-gps.c: location_gps_set_property(1651) > id [15], value [0]
04-23 16:01:44.551+0200 I/LOCATION(22613): location-gps.c: location_gps_set_property(1799) > Set prop>> requested_start_by_hybrid: 1
04-23 16:01:44.551+0200 I/LOCATION(22613): location-wps.c: location_wps_set_property(1200) > Set prop>> requested_start_by_hybrid: 1
04-23 16:01:44.551+0200 I/location(22613): es.ugr.frailty.location:iniciado servicio de localización
04-23 16:01:44.571+0200 W/AUL_AMD ( 2476): amd_status.c: __app_terminate_timer_cb(168) > send SIGKILL: No such process
04-23 16:01:44.661+0200 I/LOCATION(22613): location-gps.c: __get_availability_use_gps(508) > Check _availability use GPS
04-23 16:01:44.721+0200 W/AUL     (22613): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.servicemanager)
04-23 16:01:44.721+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:44.721+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 22613
04-23 16:01:44.731+0200 W/AUL_AMD ( 2476): amd_launch.c: __nofork_processing(1229) > __nofork_processing, cmd: 0, pid: 19708
04-23 16:01:44.731+0200 E/AUL_AMD ( 2476): amd_launch.c: __real_send(909) > send fail to client
04-23 16:01:44.731+0200 W/AUL_AMD ( 2476): amd_launch.c: __reply_handler(999) > listen fd(22) , send fd(21), pid(19708), cmd(0)
04-23 16:01:44.731+0200 W/CAPI_APPFW_APP_CONTROL(19708): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.741+0200 I/utils   (19708): specific action
04-23 16:01:44.741+0200 W/CAPI_APPFW_APP_CONTROL(19708): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.741+0200 W/CAPI_APPFW_APP_CONTROL(19708): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.741+0200 W/CAPI_APPFW_APP_CONTROL(19708): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.741+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.recorder)
04-23 16:01:44.741+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:44.741+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 19708
04-23 16:01:44.751+0200 W/AUL_AMD ( 2476): amd_launch.c: start_process(606) > child process: 22624
04-23 16:01:44.751+0200 E/RESOURCED( 2584): block.c: block_prelaunch_state(138) > insert data es.ugr.frailty.frailtylauncher, table num : 1
04-23 16:01:44.751+0200 W/AUL     (22613): launch.c: app_request_to_launchpad(298) > request cmd(0) result(19708)
04-23 16:01:44.751+0200 I/location(22613): request sent to service es.ugr.frailty.servicemanager
04-23 16:01:44.751+0200 I/location(22613): stopping es.ugr.frailty.location service
04-23 16:01:44.761+0200 E/CAPI_APPFW_APP_CONTROL(22613): app_control.c: app_control_error(138) > [app_control_add_extra_data] INVALID_PARAMETER(0xffffffea)
04-23 16:01:44.761+0200 E/location(22613): request sending failed to service es.ugr.frailty.servicemanager
04-23 16:01:44.761+0200 I/CAPI_APPFW_APPLICATION(22613): service_app_main.c: service_app_exit(441) > service_app_exit
04-23 16:01:44.791+0200 E/CAPI_APPFW_APPLICATION(22624): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:44.801+0200 E/CAPI_APPFW_APPLICATION(22624): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:44.801+0200 W/CAPI_APPFW_APP_CONTROL(22624): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.801+0200 I/utils   (22624): specific action
04-23 16:01:44.801+0200 W/CAPI_APPFW_APP_CONTROL(22624): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.801+0200 W/CAPI_APPFW_APP_CONTROL(22624): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.801+0200 W/CAPI_APPFW_APP_CONTROL(22624): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.801+0200 W/CAPI_APPFW_APP_CONTROL(22624): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:44.801+0200 W/AUL_AMD ( 2476): amd_launch.c: __send_app_launch_signal(397) > send launch signal done: 22624
04-23 16:01:44.801+0200 W/AUL     ( 2476): app_signal.c: aul_send_app_launch_request_signal(521) > aul_send_app_launch_request_signal app(es.ugr.frailty.recorder) pid(22624) type(svcapp) bg(0)
04-23 16:01:44.801+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 22
04-23 16:01:44.801+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(999) > app status : 4
04-23 16:01:44.801+0200 W/AUL_AMD ( 2476): amd_status.c: __socket_monitor_cb(1277) > (22624) was created
04-23 16:01:44.811+0200 W/STARTER ( 2701): pkg-monitor.c: _app_mgr_status_cb(395) > [_app_mgr_status_cb:395] Launch request [22624]
04-23 16:01:44.811+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(298) > request cmd(0) result(22624)
04-23 16:01:44.811+0200 I/servicemanager(19708): request sent to service es.ugr.frailty.recorder
04-23 16:01:44.811+0200 I/servicemanager(19708): requesting to save local data
04-23 16:01:44.821+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:44.821+0200 I/recorder(22624): 1 app file path: /opt/usr/apps/es.ugr.frailty.frailtylauncher/data/es.ugr.frailty.location.csv
04-23 16:01:44.821+0200 I/recorder(22624): guardando datos en local
04-23 16:01:44.831+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22624
04-23 16:01:44.831+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:44.861+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:44.871+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22624
04-23 16:01:44.871+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:44.901+0200 W/LOCATION(22613): location-gps.c: location_gps_set_property(1651) > id [15], value [0]
04-23 16:01:44.911+0200 I/LOCATION(22613): location-gps.c: location_gps_set_property(1799) > Set prop>> requested_start_by_hybrid: 0
04-23 16:01:44.911+0200 I/LOCATION(22613): location-wps.c: location_wps_set_property(1200) > Set prop>> requested_start_by_hybrid: 0
04-23 16:01:45.011+0200 W/libgps_d( 3123): GpsiHook: API: gpsStop(GpsiHookStateGps: e,c,G)
04-23 16:01:45.011+0200 W/libgps  ( 3123): [proxy__gps_stop][line = 1067]: called.
04-23 16:01:45.011+0200 W/libgps_d( 3123): GpsiHook: API: gpsStatusCb(GpsiHookStateGps: e,c,G)
04-23 16:01:45.011+0200 W/libgps_d( 3123): GpsiHook::deliverStatus() GPS_STATUS_SESSION_END
04-23 16:01:45.011+0200 W/libgps_d( 3123): GpsiHook: API: gpsStatusCb(GpsiHookStateGps: E,c,G)
04-23 16:01:45.011+0200 W/libgps_d( 3123): GpsiHookStateGps              : EXIT
04-23 16:01:45.011+0200 W/libgps_d( 3123): GpsiHookStateIdle             : ENTRY
04-23 16:01:45.011+0200 E/CAPI_LOCATION_MANAGER(19762): location_product.c: _location_manager_gps_state_cb(195) > _location_manager_gps_state_cb
04-23 16:01:45.011+0200 W/gpsd    ( 3104): HandleIncomingMessage() dest id:0, msg id:2
04-23 16:01:45.121+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:45.171+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(195) > Successfully de-registered(0)
04-23 16:01:45.171+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(295) > Successfully de-registered(0)
04-23 16:01:45.171+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(395) > Successfully de-registered(0)
04-23 16:01:45.171+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_destroy(471) > Destroy handle: 0xf736d4d0
04-23 16:01:45.231+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_create(453) > New handle created[0xf3e05c58]
04-23 16:01:45.231+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(184) > Successfully registered(1)
04-23 16:01:45.231+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(284) > Successfully registered(1)
04-23 16:01:45.231+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(384) > Successfully registered(1)
04-23 16:01:45.231+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_get_type(507) > Connected Network = 2
04-23 16:01:45.231+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:45.231+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:45.251+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.261+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.261+0200 E/LOCATION(19762): location-privacy.c: location_privacy_initialize(165) > Fail to get app_id. Err[-1]
04-23 16:01:45.261+0200 I/LOCATION(19762): location.c: location_new(269) > method: 1
04-23 16:01:45.261+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:45.261+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (gps) open success
04-23 16:01:45.261+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(remote_gps) is found
04-23 16:01:45.261+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (remote_gps) open success
04-23 16:01:45.261+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:45.261+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:45.261+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:45.261+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.281+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.281+0200 E/LOCATION(19762): location-privacy.c: location_get_app_type(54) > Fail to get app_id. Err[-1]
04-23 16:01:45.281+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:45.361+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:45.451+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:45.501+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(195) > Successfully de-registered(0)
04-23 16:01:45.501+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(295) > Successfully de-registered(0)
04-23 16:01:45.501+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(395) > Successfully de-registered(0)
04-23 16:01:45.501+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_destroy(471) > Destroy handle: 0xf3e05c58
04-23 16:01:45.561+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_create(453) > New handle created[0xf3e17d70]
04-23 16:01:45.561+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(184) > Successfully registered(1)
04-23 16:01:45.561+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(284) > Successfully registered(1)
04-23 16:01:45.561+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(384) > Successfully registered(1)
04-23 16:01:45.561+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_get_type(507) > Connected Network = 2
04-23 16:01:45.571+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:45.571+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:45.591+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.601+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.601+0200 E/LOCATION(19762): location-privacy.c: location_privacy_initialize(165) > Fail to get app_id. Err[-1]
04-23 16:01:45.601+0200 I/LOCATION(19762): location.c: location_new(269) > method: 1
04-23 16:01:45.601+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:45.601+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (gps) open success
04-23 16:01:45.601+0200 W/LOCATION(19762): module-internal.c: module_is_supported(340) > module name(remote_gps) is found
04-23 16:01:45.601+0200 W/LOCATION(19762): module-internal.c: module_new(311) > module (remote_gps) open success
04-23 16:01:45.601+0200 W/LOCATION(19762): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:45.601+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:45.601+0200 W/LOCATION(19762): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:45.611+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.611+0200 I/AUL     (19762): menu_db_util.h: _get_app_info_from_db_by_apppath(239) > path : /usr/bin/posclientd, ret : 0
04-23 16:01:45.611+0200 E/LOCATION(19762): location-privacy.c: location_get_app_type(54) > Fail to get app_id. Err[-1]
04-23 16:01:45.621+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:45.661+0200 I/SECURE_STORAGE( 2505): ss_server_ipc.c: SsServerComm(251) > write succeed
04-23 16:01:45.671+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_type_changed_callback(195) > Successfully de-registered(0)
04-23 16:01:45.671+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_ip_changed_callback(295) > Successfully de-registered(0)
04-23 16:01:45.671+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: __connection_set_proxy_changed_callback(395) > Successfully de-registered(0)
04-23 16:01:45.671+0200 I/CAPI_NETWORK_CONNECTION(19762): connection.c: connection_destroy(471) > Destroy handle: 0xf3e17d70
04-23 16:01:45.781+0200 I/servicemanager(19708): es.ugr.frailty.location alive timeout
04-23 16:01:45.781+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.location)
04-23 16:01:45.781+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:45.781+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 19708
04-23 16:01:45.791+0200 W/AUL_AMD ( 2476): amd_launch.c: __nofork_processing(1229) > __nofork_processing, cmd: 0, pid: 22613
04-23 16:01:45.791+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(298) > request cmd(0) result(22613)
04-23 16:01:45.791+0200 I/servicemanager(19708): es.ugr.frailty.location launch request sent!
04-23 16:01:45.791+0200 I/servicemanager(19708): App control destroyed.
04-23 16:01:45.821+0200 E/AUL_AMD ( 2476): amd_launch.c: __reply_handler(975) > recv error : Connection reset by peer
04-23 16:01:45.821+0200 E/AUL_AMD ( 2476): amd_launch.c: __real_send(909) > send fail to client
04-23 16:01:45.821+0200 W/AUL_AMD ( 2476): amd_launch.c: __reply_handler(999) > listen fd(22) , send fd(21), pid(22613), cmd(0)
04-23 16:01:45.841+0200 W/AUL     (22631): daemon-manager-release-agent.c: main(12) > release agent : [2:/es.ugr.frailty.location]
04-23 16:01:45.841+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 23
04-23 16:01:45.841+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 0
04-23 16:01:45.841+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(1032) > pkg_status: installed, dead pid: 22587
04-23 16:01:45.841+0200 W/AUL_AMD ( 2476): amd_request.c: __send_app_termination_signal(528) > send dead signal done
04-23 16:01:45.841+0200 I/AUL_AMD ( 2476): amd_main.c: __app_dead_handler(262) > __app_dead_handler, pid: 22587
04-23 16:01:45.841+0200 W/AUL     ( 2476): app_signal.c: aul_send_app_terminated_signal(799) > aul_send_app_terminated_signal pid(22587)
04-23 16:01:46.801+0200 W/AUL_AMD ( 2476): amd_status.c: __app_terminate_timer_cb(168) > send SIGKILL: No such process
04-23 16:01:47.341+0200 I/servicemanager(19708): es.ugr.frailty.accelerometer alive timeout
04-23 16:01:47.341+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.accelerometer)
04-23 16:01:47.341+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:47.341+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 19708
04-23 16:01:47.351+0200 E/RESOURCED( 2584): block.c: block_prelaunch_state(138) > insert data es.ugr.frailty.frailtylauncher, table num : 1
04-23 16:01:47.361+0200 W/AUL_AMD ( 2476): amd_launch.c: start_process(606) > child process: 22636
04-23 16:01:47.401+0200 E/CAPI_APPFW_APPLICATION(22636): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:47.401+0200 E/CAPI_APPFW_APPLICATION(22636): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:47.401+0200 I/utils   (22636): trying to start service: es.ugr.frailty.accelerometer
04-23 16:01:47.411+0200 W/AUL_AMD ( 2476): amd_launch.c: __send_app_launch_signal(397) > send launch signal done: 22636
04-23 16:01:47.411+0200 W/AUL     ( 2476): app_signal.c: aul_send_app_launch_request_signal(521) > aul_send_app_launch_request_signal app(es.ugr.frailty.accelerometer) pid(22636) type(svcapp) bg(0)
04-23 16:01:47.411+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(298) > request cmd(0) result(22636)
04-23 16:01:47.411+0200 I/servicemanager(19708): es.ugr.frailty.accelerometer launch request sent!
04-23 16:01:47.411+0200 I/servicemanager(19708): App control destroyed.
04-23 16:01:47.411+0200 W/STARTER ( 2701): pkg-monitor.c: _app_mgr_status_cb(395) > [_app_mgr_status_cb:395] Launch request [22636]
04-23 16:01:47.411+0200 I/utils   (22636): es.ugr.frailty.accelerometer sensor supported
04-23 16:01:47.411+0200 W/AUL_AMD ( 2476): amd_status.c: __socket_monitor_cb(1277) > (22636) was created
04-23 16:01:47.421+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:47.431+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22636
04-23 16:01:47.431+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:47.441+0200 I/utils   (22636): es.ugr.frailty.accelerometer listener started
04-23 16:01:47.441+0200 W/CAPI_APPFW_APP_CONTROL(22636): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:47.441+0200 I/utils   (22636): specific action
04-23 16:01:47.451+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:47.461+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22636
04-23 16:01:47.471+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:47.471+0200 I/accelerometer(22636): capturing data from es.ugr.frailty.accelerometer
04-23 16:01:47.481+0200 I/accelerometer(22636): es.ugr.frailty.accelerometer: SM-R760,23/04/2018,16:01:47:481,7.494320,3.933800,3.558127
04-23 16:01:47.501+0200 I/servicemanager(19708): es.ugr.frailty.gyroscope alive timeout
04-23 16:01:47.501+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.gyroscope)
04-23 16:01:47.501+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:47.501+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 19708
04-23 16:01:47.511+0200 W/AUL_AMD ( 2476): amd_launch.c: start_process(606) > child process: 22649
04-23 16:01:47.511+0200 E/RESOURCED( 2584): block.c: block_prelaunch_state(138) > insert data es.ugr.frailty.frailtylauncher, table num : 1
04-23 16:01:47.551+0200 E/CAPI_APPFW_APPLICATION(22649): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:47.551+0200 E/CAPI_APPFW_APPLICATION(22649): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:47.551+0200 I/utils   (22649): trying to start service: es.ugr.frailty.gyroscope
04-23 16:01:47.561+0200 I/utils   (22649): es.ugr.frailty.gyroscope sensor supported
04-23 16:01:47.561+0200 W/AUL_AMD ( 2476): amd_launch.c: __send_app_launch_signal(397) > send launch signal done: 22649
04-23 16:01:47.561+0200 W/AUL     ( 2476): app_signal.c: aul_send_app_launch_request_signal(521) > aul_send_app_launch_request_signal app(es.ugr.frailty.gyroscope) pid(22649) type(svcapp) bg(0)
04-23 16:01:47.561+0200 W/AUL_AMD ( 2476): amd_status.c: __socket_monitor_cb(1277) > (22649) was created
04-23 16:01:47.561+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(298) > request cmd(0) result(22649)
04-23 16:01:47.561+0200 W/STARTER ( 2701): pkg-monitor.c: _app_mgr_status_cb(395) > [_app_mgr_status_cb:395] Launch request [22649]
04-23 16:01:47.561+0200 I/servicemanager(19708): es.ugr.frailty.gyroscope launch request sent!
04-23 16:01:47.571+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:47.581+0200 I/servicemanager(19708): App control destroyed.
04-23 16:01:47.591+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22649
04-23 16:01:47.591+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:47.601+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22649
04-23 16:01:47.601+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:47.611+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:47.611+0200 I/utils   (22649): es.ugr.frailty.gyroscope listener started
04-23 16:01:47.611+0200 W/CAPI_APPFW_APP_CONTROL(22649): app_control.c: app_control_error(136) > [app_control_get_extra_data] KEY_NOT_FOUND(0xffffff82)
04-23 16:01:47.611+0200 I/utils   (22649): specific action
04-23 16:01:47.671+0200 I/accelerometer(22636): capturing data from es.ugr.frailty.accelerometer
04-23 16:01:47.671+0200 I/accelerometer(22636): es.ugr.frailty.accelerometer: SM-R760,23/04/2018,16:01:47:675,8.207381,4.993821,3.290131
04-23 16:01:47.671+0200 I/gyroscope(22649): capturing data from es.ugr.frailty.gyroscope
04-23 16:01:47.671+0200 I/gyroscope(22649): es.ugr.frailty.gyroscope: SM-R760,23/04/2018,16:01:47:679,-60.970001,-11.200000,8.540000
04-23 16:01:47.861+0200 I/accelerometer(22636): capturing data from es.ugr.frailty.accelerometer
04-23 16:01:47.861+0200 I/accelerometer(22636): es.ugr.frailty.accelerometer: SM-R760,23/04/2018,16:01:47:865,9.107083,2.002793,2.445465
04-23 16:01:47.871+0200 I/gyroscope(22649): capturing data from es.ugr.frailty.gyroscope
04-23 16:01:47.871+0200 I/gyroscope(22649): es.ugr.frailty.gyroscope: SM-R760,23/04/2018,16:01:47:876,-103.180000,-29.190001,9.380000
04-23 16:01:47.881+0200 I/servicemanager(19708): es.ugr.frailty.location sleep timeout
04-23 16:01:47.881+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(284) > request cmd(0) to(es.ugr.frailty.location)
04-23 16:01:47.881+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 0
04-23 16:01:47.881+0200 W/AUL_AMD ( 2476): amd_launch.c: _start_app(1782) > caller pid : 19708
04-23 16:01:47.891+0200 W/AUL_AMD ( 2476): amd_launch.c: start_process(606) > child process: 22663
04-23 16:01:47.901+0200 E/RESOURCED( 2584): block.c: block_prelaunch_state(138) > insert data es.ugr.frailty.frailtylauncher, table num : 1
04-23 16:01:47.951+0200 W/AUL_AMD ( 2476): amd_launch.c: __send_app_launch_signal(397) > send launch signal done: 22663
04-23 16:01:47.951+0200 W/AUL     ( 2476): app_signal.c: aul_send_app_launch_request_signal(521) > aul_send_app_launch_request_signal app(es.ugr.frailty.location) pid(22663) type(svcapp) bg(0)
04-23 16:01:47.951+0200 W/STARTER ( 2701): pkg-monitor.c: _app_mgr_status_cb(395) > [_app_mgr_status_cb:395] Launch request [22663]
04-23 16:01:47.961+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:47.971+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22663
04-23 16:01:47.971+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:47.981+0200 W/AUL     (19708): launch.c: app_request_to_launchpad(298) > request cmd(0) result(22663)
04-23 16:01:47.981+0200 I/servicemanager(19708): es.ugr.frailty.location stop request sent!
04-23 16:01:47.981+0200 I/servicemanager(19708): App control destroyed.
04-23 16:01:47.991+0200 E/CAPI_APPFW_APPLICATION(22663): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:47.991+0200 E/CAPI_APPFW_APPLICATION(22663): service_app_error.c: service_app_error(67) > [service_app_add_event_handler] INVALID_PARAMETER(0xffffffea)
04-23 16:01:48.001+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 14
04-23 16:01:48.001+0200 W/AUL_AMD ( 2476): amd_request.c: __send_result_to_client(91) > __send_result_to_client, pid: 22663
04-23 16:01:48.011+0200 W/AUL_AMD ( 2476): amd_status.c: __socket_monitor_cb(1277) > (22663) was created
04-23 16:01:48.011+0200 W/AUL_AMD ( 2476): amd_request.c: __request_handler(669) > __request_handler: 12
04-23 16:01:48.031+0200 W/LOCATION(22663): location-common-util.c: location_is_onboard_gps_use_only(180) > app_type [capp]
04-23 16:01:48.031+0200 E/LOCATION(22663): location-common-util.c: location_is_onboard_gps_use_only(191) > Failed to get metadata. Err[-2]
04-23 16:01:48.031+0200 E/PKGMGR_INFO(22663): pkgmgrinfo_appinfo.c: pkgmgrinfo_appinfo_destroy_appinfo(3554) > (handle == NULL) appinfo handle is NULL
04-23 16:01:48.041+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:48.061+0200 I/gyroscope(22649): capturing data from es.ugr.frailty.gyroscope
04-23 16:01:48.061+0200 I/gyroscope(22649): es.ugr.frailty.gyroscope: SM-R760,23/04/2018,16:01:48:65,26.389999,-23.309999,56.840000
04-23 16:01:48.061+0200 I/accelerometer(22636): capturing data from es.ugr.frailty.accelerometer
04-23 16:01:48.071+0200 I/accelerometer(22636): es.ugr.frailty.accelerometer: SM-R760,23/04/2018,16:01:48:71,8.650054,1.780260,2.349752
04-23 16:01:48.071+0200 I/LOCATION(22663): location.c: location_new(269) > method: 0
04-23 16:01:48.071+0200 W/LOCATION(22663): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:48.081+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:48.081+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(gps) is found
04-23 16:01:48.081+0200 W/LOCATION(22663): module-internal.c: module_new(311) > module (gps) open success
04-23 16:01:48.091+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(remote_gps) is found
04-23 16:01:48.091+0200 W/LOCATION(22663): module-internal.c: module_new(311) > module (remote_gps) open success
04-23 16:01:48.091+0200 W/LOCATION(22663): location-common-util.c: location_is_onboard_gps_use_only(149) > onboard-gps-use-only mode: [0]
04-23 16:01:48.101+0200 W/LOCATION(22663): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:48.101+0200 W/LOCATION(22663): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:48.131+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(wps) is found
04-23 16:01:48.131+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(wps) is found
04-23 16:01:48.131+0200 W/LOCATION(22663): module-internal.c: module_new(311) > module (wps) open success
04-23 16:01:48.151+0200 W/LOCATION(22663): module-internal.c: module_is_supported(340) > module name(remote_wps) is found
04-23 16:01:48.151+0200 W/LOCATION(22663): module-internal.c: module_new(311) > module (remote_wps) open success
04-23 16:01:48.151+0200 W/LOCATION(22663): location-wps.c: __update_remote_connection_status(215) > update FALSE
04-23 16:01:48.151+0200 W/LOCATION(22663): location-wps.c: __update_remote_connection_status(220) > is_remote_connected[0]
04-23 16:01:48.171+0200 W/LOCATION(22663): location-gps.c: location_gps_set_property(1651) > id [14], value [0]
04-23 16:01:48.171+0200 I/LOCATION(22663): location-gps.c: location_gps_set_property(1778) > Set prop>> hybrid_obj: 0xf7979c18
04-23 16:01:48.171+0200 I/LOCATION(22663): location-gps.c: location_gps_set_property(1793) > Set prop>> is_hybrid: 1
04-23 16:01:48.171+0200 I/LOCATION(22663): location-wps.c: location_wps_set_property(1182) > Set prop>> hybrid_obj: 0xf7979c18
04-23 16:01:48.171+0200 I/LOCATION(22663): location-wps.c: location_wps_set_property(1194) > Set prop>> is_hybrid: 1
04-23 16:01:48.171+0200 I/location(22663): es.ugr.frailty.location: creado servicio de localización
04-23 16:01:48.251+0200 I/LOCATION(22663): location-hybrid.c: location_hybrid_start(641) > IOS Connected [0]
04-23 16:01:48.261+0200 I/gyroscope(22649): capturing data from es.ugr.frailty.gyroscope
04-23 16:01:48.261+0200 I/gyroscope(22649): es.ugr.frailty.gyroscope: SM-R760,23/04/2018,16:01:48:265,-29.889999,-3.500000,-23.940001
04-23 16:01:48.261+0200 I/accelerometer(22636): capturing data from es.ugr.frailty.accelerometer
04-23 16:01:48.271+0200 I/accelerometer(22636): es.ugr.frailty.accelerometer: SM-R760,23/04/2018,16:01:48:272,7.661818,0.122034,6.745367
04-23 16:01:48.321+0200 W/LOCATION(22663): location-gps.c: __update_remote_connection_status(362) > update FALSE
04-23 16:01:48.321+0200 W/LOCATION(22663): location-gps.c: __update_remote_connection_status(367) > is_remote_connected[0]
04-23 16:01:48.321+0200 I/LOCATION(22663): location-gps.c: __get_availability_use_gps(508) > Check _availability use GPS
04-23 16:01:48.411+0200 W/CRASH_MANAGER(21562): worker.c: worker_job(1205) > 0621185726563152449210
